### PR TITLE
Fix for upload stopping when the device is rotated (#255)

### PR DESCRIPTION
**Pull request by sd1998**
Used DialogFragment to fix Issue #255

### CODE REVIEW

File: src/main/java/org/amahi/anywhere/activity/ServerFilesActivity.java [hunk 1]
```diff
@@ -64,6 +65,7 @@
 import org.amahi.anywhere.util.Fragments;
 import org.amahi.anywhere.util.Intents;
 import org.amahi.anywhere.util.Mimes;
+import org.videolan.libvlc.Dialog;
```
Comment (csoni111) [inline src/main/java/org/amahi/anywhere/activity/ServerFilesActivity.java]: Also please remove unnecessary imports and reformat the code.

File: src/main/java/org/amahi/anywhere/activity/ServerFilesActivity.java [hunk 2]
```diff
@@ -555,4 +553,22 @@ private void tearDownFilesState(Bundle state) {
         private State() {
         }
     }
+
+    @SuppressLint("ValidFragment")
```
Comment (csoni111) [inline src/main/java/org/amahi/anywhere/activity/ServerFilesActivity.java]: Why did you suppress the lint? It's giving a fair warning:
image

Also the app **crashes** on screen rotation with this in the logs:
` java.lang.RuntimeException: Unable to start activity ComponentInfo{org.amahi.anywhere/org.amahi.anywhere.activity.ServerFilesActivity}: android.app.Fragment$InstantiationException: Unable to instantiate fragment org.amahi.anywhere.activity.ServerFilesActivity$ProgressDialogFragment: make sure class name exists, is public, and has an empty constructor that is public.`

**Comment by cpg** [comment]
@sd1998 any updates?

**Comment by sd1998** [comment]
@cpg I could not concentrate on this due to work load in college I will be working on it now

**Pull request closed without merging**